evaluationcontext.go

```diff
@@ -1,26 +1,25 @@
 package flagsmith
 
+// EvaluationContext is contextual data used during feature flag evaluation.
+//
+// The zero value represents the current Flagsmith environment.
 type EvaluationContext struct {
-	Environment *EnvironmentEvaluationContext `json:"environment,omitempty"`
-	Feature     *FeatureEvaluationContext     `json:"feature,omitempty"`
-	Identity    *IdentityEvaluationContext    `json:"identity,omitempty"`
+	identifier string
+	traits     map[string]interface{}
 }
 
-type EnvironmentEvaluationContext struct {
```

Comment:
Why did we get rid of this?

Reply:
IMO it doesn't provide any value to the SDK, and we should encourage customers to create evaluation contexts using our functions such as `NewEvaluationContext`. I can't see any reason why it should have been here in the first place. I'm happy to add it back if we can find a good reason to.

Reply:
It was used to dynamically pass environment key to get flags etc